src/svd/clusterinfo.rs

```diff
@@ -7,22 +7,18 @@ use types::Parse;
 use encode::Encode;
 #[cfg(feature = "unproven")]
 use new_element;
-use parse;
 
 use error::SVDError;
-use svd::access::Access;
 use svd::registercluster::RegisterCluster;
+use svd::defaults::Defaults;
 
 #[derive(Clone, Debug, PartialEq)]
 pub struct ClusterInfo {
     pub name: String,
     pub description: String,
     pub header_struct_name: Option<String>,
     pub address_offset: u32,
-    pub size: Option<u32>,
-    pub access: Option<Access>,
-    pub reset_value: Option<u32>,
-    pub reset_mask: Option<u32>,
+    pub defaults: Defaults,
```

Comment:
(as discussed earlier) I'd rather use `register_properties` or `default_register_properties` here to express the meaning of these "defaults". Note that this property set is called `registerPropertiesGroup` in SVD spec.

Reply:
Done